src/test/java/sirius/kernel/nls/NLSSpec.groovy

```diff
@@ -148,6 +144,26 @@ class NLSSpec extends BaseSpecification {
         NLS.parseUserString(Amount.class, input).toString() == "34,54"
     }
 
+    def "parseUserString works for integers"() {
+        expect:
+        NLS.parseUserString(Integer.class, input) == output
+
+        where:
+        input     | output
+        "42"      | 42
+        "77,0000" | 77
```

Comment:
ensure that german and english 77.000 and 77,000 still yield 77000 and also other edge cases